integration_tests/cairozero_test.go

```diff
@@ -127,10 +133,73 @@ func TestCairoZeroFiles(t *testing.T) {
 		}
 	}
 
+	WriteBenchMarksToFile(benchmarkMap)
+
 	clean(root1)
 	clean(root2)
 }
 
+// Save the Benchmarks for the integration tests in `BenchMarks.txt`
+func WriteBenchMarksToFile(benchmarkMap map[string][2]int) {
+	headers := []string{"File", "PythonVM (ms)", "GoVM (ms)"}
+	columnWidths := []int{33, 15, 10}
+
+	totalWidth := 0
+	for _, width := range columnWidths {
+		totalWidth += width + 3
+	}
+
+	totalWidth += 1
+
+	border := strings.Repeat("=", totalWidth)
+
+	var sb strings.Builder
+
+	sb.WriteString(border + "\n")
+	sb.WriteString(formatRow(headers, columnWidths) + "\n")
+	sb.WriteString(border + "\n")
+
+	for key, values := range benchmarkMap {
+		row := []string{key}
+		for _, value := range values {
+			row = append(row, strconv.Itoa(value))
+		}
+		sb.WriteString(formatRow(row, columnWidths) + "\n")
+	}
+
+	sb.WriteString(border + "\n")
+
+	fileName := "BenchMarks.txt"
+	file, err := os.Create(fileName)
+	if err != nil {
+		fmt.Println("Error creating file: ", err)
+		return
+	}
+	defer file.Close()
+
+	_, err = file.WriteString(sb.String())
+	if err != nil {
+		fmt.Println("Error writing to file: ", err)
+	} else {
+		fmt.Println("Benchmarks successfully written to:", fileName)
+	}
+}
+
+// format a row with borders and spaces based on the column widths
+func formatRow(row []string, widths []int) string {
+	var sb strings.Builder
+	sb.WriteString("|")
+	for i, col := range row {
+		sb.WriteString(" " + padRight(col, widths[i]) + " |")
+	}
+	return sb.String()
+}
```

Comment:
Check out: https://pkg.go.dev/text/tabwriter
Might be a lot easier and its in the stdlib.

Reply:
I don't think we can achieve different column width for the same row with tabwriter. Can we go ahead with the current impl ?

Reply:
Updated with tabwriter. Thanks for the suggestion.